lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -635,6 +635,40 @@ class SqueezeOpConversionPattern : public OpConversionPattern<ttir::SqueezeOp> {
   }
 };
 
+class BroadcastOpConversionPattern
```

Comment:
I've noticed that broadcast doesn't have verifier can you add it?

Reply:
Added